packages/amplify-category-geo/src/service-walkthroughs/mapWalkthrough.ts

```diff
@@ -121,14 +109,13 @@ export const mapStyleWalkthrough = async (parameters: Partial<MapParameters>): P
     const mapStyleDefault = parameters.dataProvider && parameters.mapStyleType ?
         getGeoMapStyle(parameters.dataProvider, parameters.mapStyleType) : 'VectorEsriStreets';
 
-    const mapStyleTypePrompt = {
-        type: 'list',
-        name: 'mapStyle',
-        message: `Specify the map style. Refer ${apiDocs.mapStyles}`,
-        choices: mapStyleChoices,
-        default: mapStyleDefault
-    };
-    return getMapStyleComponents((await inquirer.prompt([mapStyleTypePrompt])).mapStyle as MapStyle);
+    const mapStyleDefaultIndex = mapStyleChoices.findIndex(item => item.value === mapStyleDefault);
+    const mapStyleInput = await prompter.pick<'one', string>(
```

Comment:
nit: you should be able to omit the `<'one', string>` typedef as that's the default

Reply:
if the `value` in the choices is an enum, this throws an error unless we explicitly specify it as `string`. Changed in places where it is actually string.